.github/Contributing.md

```diff
@@ -32,6 +32,8 @@ cd a32nx
 
 Note that you should use `run.sh` instead of `run.cmd` if you are on Linux (including WSL).
 
+Also, if using WSL, make sure that the `Vmmem` process is not memory limited, or has at least `10GB` of memory. This is configured in `C:\<user>\.wslconfig`
```

Comment:
> Also, if using WSL, make sure that the `Vmmem` process is not memory limited, or has at least `10GB` of memory. This is configured in `C:\<user>\.wslconfig`

Change to

If you are using WSL, ensure that the `Vmmem` process is not memory limited. At least `10GB` of memory is the recommended setting. This can be configured in `C:\<user>\.wslconfig`.